tools/generate_conda_file.py

```diff
@@ -84,7 +84,7 @@
     "gensim": "gensim>=3.7.0",
     "nltk": "nltk>=3.4",
     "seqeval": "seqeval>=0.0.12",
-    "bertsum": "--editable=git+https://github.com/daden-ms/BertSum.git",
+    "bertsum": "git+https://github.com/daden-ms/BertSum.git#egg=src",
```

Comment:
have you tested  this? I tried "pip install git+https://github.com/daden-ms/BertSum.git#egg=src" , it gives me error message:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-lacg6bil/bertsum/setup.py'
  ----------------------------------------
  ERROR: Failed building wheel for bertsum

Reply:
According to https://pip.pypa.io/en/stable/reference/pip_install/#git, the egg string should be the project name
How about this: 
git+https://github.com/daden-ms/BertSum.git@030c139c97bc57d0c31f6515b8bf9649f999a443#egg=BertSum   ?

Reply:
Yeah, I've tested what I committed and it worked fine. I also tested your version it also worked, but is it tied to a specific commit (because of 030c139c97bc57d0c31f6515b8bf9649f999a443)? What if you make changes to your fork later? 
I also tried  git+https://github.com/daden-ms/BertSum.git#egg=BertSum and it worked too, so let's use this instead.

Reply:
Actually I want to tie it to a specific commit so we know which exact version it works with. If we need to make the change, we just need change the conda file again. It's just like the version for the other packages.

Reply:
OK. That's fine by me. Just updated it.